dc-cudami-frontend/cudami-frontend-website/src/main/java/de/digitalcollections/cudami/frontend/website/controller/MainController.java

```diff
@@ -0,0 +1,63 @@
+package de.digitalcollections.cudami.frontend.website.controller;
+
+import de.digitalcollections.cudami.frontend.website.config.TemplateConfig;
+import de.digitalcollections.cudami.frontend.website.service.ContentService;
+import de.digitalcollections.model.identifiable.web.Webpage;
+import java.lang.invoke.MethodHandles;
+import java.util.Locale;
+import java.util.UUID;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.server.ResponseStatusException;
+
+@Controller
+/** see GlobalControllerAdvice for global model objects */
+public class MainController {
+
+  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
wenn static final, bitte in Großbuchstaben setzen

Reply:
erwischt: habe ich hier zum ersten mal so gemacht wie DZP-WFS.... ist wohl ein unendlicher Religionskrieg. ich mache es gerne groß (wie bisher), ist mein favorit :+1: